core/src/test/java/com/linecorp/armeria/internal/common/stream/RecoverableStreamMessageTest.java

```diff
@@ -74,6 +76,50 @@ void resumeOnError() {
         assertThat(recoverable.collect().join()).contains(1, 2, 3, 4, 5, 6);
     }
 
+    @Test
```

Comment:
Could we also add the tests for chained `recover()` and/or `recoverAndResume()` calls, including the mixture of `recover` and `recoverAndResume`?

Reply:
of course! 😀